src/database/vehicleDb.ts

```diff
@@ -0,0 +1,23 @@
+import { dbConnection } from './dbConnection';
+import { Vehicle } from '../interfaces/Vehicle';
+
+export const vehicleDb = {
+  async get(vrm: string): Promise<Vehicle> {
```

Comment:
If this is the function that gets those fields for the main query it won't want to be Async

Reply:
Where it's called, it's awaited so we have the result before it moves on to the insert.
![image](https://user-images.githubusercontent.com/13391709/188467055-a888ff0f-36b2-478e-9f64-94c20d2b387a.png)

Reply:
Thanks, I was looking for this line but couldn't find it